include/seqan3/range/views/minimiser.hpp

```diff
@@ -45,7 +46,7 @@ class minimiser_view : public std::ranges::view_interface<minimiser_view<urng_t>
     urng_t urange;
 
     //!\brief The number of values in one window.
-    uint32_t window_values_size;
+    size_t window_values_size;
```

Comment:
why?

Reply:
This is a size and we should use the type for it, `size_t`.